### PR TITLE
Tlmblgis 161

### PR DESCRIPTION
**Pull request by dovek**
Due: 10/4
@dlhaines 
@pushyamig

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,39 +1,34 @@
-FROM ubuntu:14.04
-
+FROM ubuntu:16:04
 MAINTAINER Kyle Dove <dovek@umich.edu>
```
Comment (dlhaines) [inline Dockerfile:2]: Probably shouldn't be a specific person since sometimes they move on to "better" things.

File: Dockerfile [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
 
-# TODO: Take out the work directory change here. Suggested directory: /usr/apps/Ocellus/
-# Install Ocellus
-WORKDIR /tmp/
+# install dependencies and link node to nodejs, as ubuntu doesn't do this by default.
+RUN apt-get install -y nodejs curl libmysqlclient-dev python-dev gunicorn python-pip django-filter python-ldap npm git libssl-dev libffi-dev xmlsec1 && \
+	ln -s /usr/bin/nodejs /usr/bin/node
 
-COPY . /tmp/
+# install depedendent python packages that aren't available as pre-built libraries
```
Comment (dlhaines) [inline Dockerfile]: spelling

**Comment by dlhaines** [comment]
Not a comment about this project in particular, but how are we to make sense of a Docker file written by someone else a year later when it needs maintenance?

**Comment by dlhaines** [comment]
How does this optimize  the Docker file?

**Comment by pushyamig** [comment]
Looks Good!

**Pull request merged**